src/wp-includes/class-wp-scripts.php

```diff
@@ -295,14 +295,48 @@ public function do_item( $handle, $group = false ) {
 		}
 
 		$before_handle = $this->print_inline_script( $handle, 'before', false );
-		$after_handle  = $this->print_inline_script( $handle, 'after', false );
 
 		if ( $before_handle ) {
 			$before_handle = sprintf( "<script%s id='%s-js-before'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $before_handle );
 		}
 
-		if ( $after_handle ) {
-			$after_handle = sprintf( "<script%s id='%s-js-after'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $after_handle );
+		$strategy = $this->get_eligible_loading_strategy( $handle );
+
+		$after_handle = '';
+
+		// Eligible loading strategies will only be 'async', 'defer', or ''.
```

Comment:
What about `blocking`?

Reply:
@westonruter this is not where we define a strategy, but where we get the value back from `get_eligible_loading_strategy()`. If the intended strategy (i.e the one supplied in `$args`) was `'blocking'` that still gets returned as `''` and we treat that as blocking.

Reply:
Even if the intended strategy was set as `''` in `$args`, that too is treated as the default strategy, which is blocking.